styles/controllers/grid/grid.less

```diff
@@ -329,8 +329,9 @@
 	.pkp_linkaction_removeUserGroup,
 	.pkp_linkaction_deleteAuthor,
 	.pkp_linkaction_deleteQuery,
-	.pkp_linkaction_deleteChapter {
-		color: @offset;
+	.pkp_linkaction_deleteChapter,
```

Comment:
(This is maybe better moved into the application `grid.less`, since it's OMP-specific.)